### PR TITLE
[FIX] account_credit_control: missing permission

### PR DESCRIPTION
**Pull request by cubells**
User with Accounting & Finance permissions got an error if he doesn't belongs to a Credit Control group on viewing partner form.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check also the groups (or the lack of) associated to the created views, in `res.partner`.

**Comment by cubells** [comment]
I think It's done now.

**Comment by pedrobaeza** [comment]
Remove then groups of the field.

File: account_credit_control/security/ir.model.access.csv [hunk 1]
```diff
@@ -18,3 +18,6 @@ account_credit_control.ir_model_access_289,credit_control_info_level,account_cre
 account_credit_control.ir_model_access_290,credit_control_fin_user_line,account_credit_control.model_credit_control_line,account.group_account_user,1,0,0,0
 account_credit_control.ir_model_access_291,credit_control_fin_invoice_line,account_credit_control.model_credit_control_line,account.group_account_invoice,1,0,0,0
 account_credit_control.ir_model_access_292,credit_control_fin_manager_line,account_credit_control.model_credit_control_line,account.group_account_manager,1,1,1,1
+account_credit_control.ir_model_access_293,credit_control_fin_user_policy,account_credit_control.model_credit_control_policy,account.group_account_user,1,0,0,0
+account_credit_control.ir_model_access_294,credit_control_fin_invoice_policy,account_credit_control.model_credit_control_policy,account.group_account_invoice,1,0,0,0
+account_credit_control.ir_model_access_295,credit_control_fin_manager_policy,account_credit_control.model_credit_control_policy,account.group_account_manager,1,1,1,1
```
Comment (pedrobaeza) [inline account_credit_control/security/ir.model.access.csv]: The idea here is that you have the permissions for credit control policy isolated from accounting permissions, so I don't see this permission correct. You should instead followed a similar strategy that the one of the view: limiting views to those that have the correct permission.

Comment (cubells) [reply account_credit_control/security/ir.model.access.csv]: Yes, I think so. This lines added are not necessary.

I'll remove tomorrow.

**Comment by pedrobaeza** [comment]
And remember to change module version number

**Comment by pedrobaeza** [comment]
I still find a view with groups assigned in the incorrect place (fields instead of the full view): https://github.com/OCA/account-financial-tools/blob/9.0/account_credit_control/views/account_invoice.xml

**Comment by cubells** [comment]
Done!

**Review by pedrobaeza — APPROVED** [review]
@chienandalu please review

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@cubells, please port this to v10.

**Pull request merged**